unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,70 @@
+type Property {
+  properties
```

Comment:
is this so that a property can have child properties?

Reply:
Oh, yes it was for this reason - based on the old schema. But recalling our previous discussion we probably don't need it anymore, will remove.